src/main/java/com/stripe/model/ExpandableField.java

```diff
@@ -0,0 +1,29 @@
+package com.stripe.model;
+
+//TODO: Should this extend StripeObject?
```

Comment:
Hm, any particular reason that it should? I can't see that as being obviously desirable.